### ISSUE TITLE
Update accessibility part in User input

### ISSUE THREAD

**Issue by tidoust** [w3c/web-roadmaps#184]
In particular, [WCAG 2.1](https://www.w3.org/TR/WCAG21/) addresses more accessibility requirements for mobile accessibility that should be highlighted.

**Comment by tidoust** [comment]
I prepared a pull request to adjust the text that mentions accessibility. The proposed updates include:
- renaming the "User Input" page into "User Interaction", which seems better and broader
- mentioning accessibility in the description of the page that appears on the roadmap's home page
- adding an introduction to the page to set the scope (and thus talk about accessibility)
- referencing more guidelines and pointing out mobile-specific ones when they exist

@xfq, I would suggest to merge that pull request after your review and then get back to the accessibility team with the resulting document (so that they can more easily review the document).

**Comment by tidoust** [comment]
Closing the issue as text has been updated according to feedback received. More specific issues may be created as needed.
